go.mod

```diff
@@ -35,19 +35,15 @@ require (
 	github.com/ipfs/go-ipld-cbor v0.0.5
 	github.com/ipfs/go-ipld-format v0.2.0
 	github.com/ipfs/go-log v1.0.4
-	github.com/ipfs/go-log/v2 v2.1.1
+	github.com/ipfs/go-log/v2 v2.1.3
 	github.com/ipfs/go-merkledag v0.3.2
-	github.com/libp2p/go-libp2p v0.12.0
+	github.com/libp2p/go-libp2p v0.14.0
```

Comment:
Roman told me this v0.14.0 solved many simultaneous connections issues for Anytype, so I tried here. Didn't improve significantly, but not bad to have I suppose?